src/main/java/com/google/maps/model/AddressType.java

```diff
@@ -15,15 +15,20 @@
 
 package com.google.maps.model;
 
-import com.google.maps.internal.StringJoin.UrlValue;
-
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+import com.google.maps.internal.StringJoin.UrlValue;
+
 /**
  * The Adress types. Please see
  * <a href="https://developers.google.com/maps/documentation/geocoding/#Types">Address
  * Types</a> for more detail.
+ * Some address contains additional place category. Please see 
```

Comment:
"Some addresses contain additional place categories."

While you're in here, would you mind fixing the type on line 27 too? "The Adress types" should read "The *address* types"